lib/node_modules/@stdlib/stats/base/dists/triangular/median/README.md

```diff
@@ -141,6 +141,95 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/triangular/median.h"
+```
+
+#### stdlib_base_dists_triangular_median( a, b, c)
+
+Evaluates the [median][median] for a [triangular][triangular-distribution] distribution with parameters `a` (lower limit), `b` (upper limit) and `c` (mode).
+
+```c
+double out = stdlib_base_dists_triangular_median( 0.0, 1.0, 0.8 );
+// returns ~0.632
+```
+
+The function accepts the following arguments:
+
+-   **a**: `[in] double` lower limit.
+-   **b**: `[in] double` upper limit.
+-   **c**: `[in] double` mode.
+
+```c
+double stdlib_base_dists_triangular_median( const double a, const double b, const double c );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/stats/base/dists/triangular/median.h"
+#include <stdlib.h>
+#include <stdio.h>
+
```

Comment:
Please define the below here 

```c
static double random_uniform( const double min, const double max ) {
    double v = (double)rand() / ( (double)RAND_MAX + 1.0 );
    return min + ( v*(max-min) );
}
```

and use it to draw the random values for `a`, `b`, and `c` similar to the changes made in https://github.com/stdlib-js/stdlib/pull/3354/commits/da14b77af9c95ec5185a6aced49030f53d97fdee.